guides/spin/cli/index.md

```diff
@@ -71,4 +71,29 @@ and the [README](https://github.com/spinnaker/spin/blob/master/README.md) for mo
 
 ### OAuth2
 
-Coming soon in Spinnaker 1.9.x.
+`spin` can be configured with OAuth2.0 to authenticate calls against Spinnaker. The configuration
+block looks like this:
+
+```yaml
+auth:
+  enabled: true
+  oauth2:
+    authUrl: # OAuth2 provider auth url
+    tokenUrl: # OAuth2 provider token url
+    clientId: # OAuth2 client id
+    clientSecret: # OAuth2 client secret
+    scopes: # Scopes requested for the token
+    - scope1
+    - scope2
+```
+
+See https://www.spinnaker.io/setup/security/authentication/oauth/providers/ 
+to see examples for acquiring a clientId/clientSecret from your provider.
+
+Unlike X.509, OAuth2 needs initialized once to authenticate with the provider before
+it can be used for automation. To authenticate, configure OAuth2 as shown and execute
+any `spin` command. You will be prompted to authenticate with your OAuth2 provider
+and paste an access code in the terminal the `spin` command is executing in. `spin`
+then exchanges the code for an OAuth2 token, which it caches for future use. All
+subsequent `spin` calls will use the cached OAuth2 token for authentication with
```

Comment:
What happens when your token expires?

Reply:
Can you share how once this is done, copying the `~/.spin/config` file to another server allows it be used for w/o intervention by some automated system?